app/src/main/java/Script/RealScriptProvider.java

```diff
@@ -22,59 +20,73 @@
 import javax.xml.parsers.DocumentBuilderFactory;
 import javax.xml.parsers.ParserConfigurationException;
 
-public class RealScriptProvider implements IScriptProvider {
-	
-	String _path;
-	List<BookData> Books = new ArrayList<BookData>();
+public class RealScriptProvider implements IScriptProvider
+{
+	String path;
+	ArrayList<BookData> books = new ArrayList<BookData>();
```

Comment:
Is there a particular reason to make the declaration more specific? See e.g. http://stackoverflow.com/questions/2279030/type-list-vs-type-arraylist-in-java